library/src/test/scala/SchemaExample.scala

```diff
@@ -632,6 +632,138 @@ object NewSchema {
           |    }
           |}""".stripMargin)
 
+  val completeExampleCodeSerializer =
+    """package com.example
+      |import _root_.sjsonnew._
+      |import _root_.sjsonnew.BasicJsonProtocol._
```

Comment:
That's how its done now.

For instance, for a simple schema that just defines an enumeration, we generate: 

```scala
import _root_.sjsonnew.{ deserializationError, serializationError, Builder, JsonFormat, Unbuilder }
trait simpleEnumerationExampleFormat { self: sbt.datatype.StringFormat =>
  implicit lazy val simpleEnumerationExampleFormat: JsonFormat[simpleEnumerationExample] = new JsonFormat[simpleEnumerationExample] {
    override def read[J](jsOpt: Option[J], unbuilder: Unbuilder[J]): simpleEnumerationExample = {
      jsOpt match {
        case Some(js) =>
          unbuilder.readString(js) match {
            case "first" => simpleEnumerationExample.first
            case "second" => simpleEnumerationExample.second
          }
        case None =>
          deserializationError("Expected JsString but found None")
      }
    }
    override def write[J](obj: simpleEnumerationExample, builder: Builder[J]): Unit = {
      val str = obj match {
        case simpleEnumerationExample.first => "first"
        case simpleEnumerationExample.second => "second"
      }
      builder.writeString(str)
    }
  }
}

trait Codec { self: _root_.simpleEnumerationExampleFormat with sbt.datatype.StringFormat => }
object Codec extends Codec with _root_.simpleEnumerationExampleFormat with sbt.datatype.StringFormat
```